integrations/hubspot/client.go

```diff
@@ -63,9 +65,39 @@ func connStatus(i *integration) sdkintegrations.OptFn {
 	})
 }
 
-// TODO(INT-105): Implement
-func connTest(_ *integration) sdkintegrations.OptFn {
+// connTest is an optional connection test provided by the integration
+// to AutoKitteh. It is used to verify that the connection is working
+// as expected. The possible results are "OK" and "error".
+func connTest(i *integration) sdkintegrations.OptFn {
 	return sdkintegrations.WithConnectionTest(func(ctx context.Context, cid sdktypes.ConnectionID) (sdktypes.Status, error) {
+		if !cid.IsValid() {
+			return sdktypes.NewStatus(sdktypes.StatusCodeError, "Init required"), nil
+		}
+
+		vs, err := i.vars.Get(ctx, sdktypes.NewVarScopeID(cid))
+		if err != nil {
+			return sdktypes.InvalidStatus, err
+		}
+
+		token := vs.GetValueByString("oauth_AccessToken")
```

Comment:
This won't work because the access token stored when you create the connection is effective only for 30 minutes. You need to read `oauth_RefreshToken` and generate a fresh access token with it. BTW, this would be a sufficient test (no need to run any API call with the access token) because HubSpot won't given you a token if the refresh token is missing/invalid.

Reply:
committed a fix